tests/acceptance/features/bootstrap/SearchContext.php

```diff
@@ -39,19 +39,22 @@ class SearchContext implements Context {
 	 * @When user :user searches for :pattern and limits the results to :limit items using the WebDAV API
 	 * @When user :user searches for :pattern using the WebDAV API requesting these properties:
 	 * @When user :user searches for :pattern and limits the results to :limit items using the WebDAV API requesting these properties:
+	 * @When user :user searches for :pattern inside :scope using the WebDAV API
 	 *
 	 * @param string $user
 	 * @param string $pattern
 	 * @param string|null $limit
 	 * @param TableNode|null $properties
+	 * @param string|null $scope
 	 *
 	 * @return void
 	 */
 	public function userSearchesUsingWebDavAPI(
 		string    $user,
 		string  $pattern,
 		?string	  $limit = null,
-		TableNode $properties = null
+		TableNode $properties = null,
+		?string $scope = null
```

Comment:
Let data table arg be the last one
```suggestion
		?string $scope = null,
		TableNode $properties = null
```